Active Members host at least one Technical Seminar (#286)
ComputerScienceHouse/Constitution: constitution.tex

% Formatted as term | definition
\begin{tabular}{l l}

> 	Technical                  & \textit{Relevant to computing or electronics} \\

Reviewer: Did you intentionally not make this reflect the updated definition of Technical Seminar?

Author: No, I forgot, my branch was out of date (hence same PR)

> 	Technical Seminar          & \textit{A seminar containing Technical practical knowledge} \\

Reviewer: Do we need to define Techical and Technical Seminar? I feel like we could just use the dictionary definition of seminar with our definition of Technical

Also, this phrasing just sounds wrong. I think we should define Technical as basically what the current technical seminar definition is (minus the seminar part) and remove the definition of Technical Seminar

Author: Technical should be "practical knowledge relevant to computing or electronics"?

Author: I agree with removing Technical Seminar and now we have Technical seminars. I'll edit

\asubsubsection{Expectations of Active Members}
Active Members are required to pay dues as stated in \ref{Collection of Dues}, attend all House Meetings and E-Board candidate speeches, and attend at least fifteen of the directorship meetings for each semester in which they are an Active Member.
They are expected to sign a copy of the Membership Agreement.

> Active Members must host at least one Technical Seminar.

Reviewer 2: Because this goes after the first sentence (where at the end of the sentence it defines the time restraints) this should have a time attached to it, semesterly or yearly

Reviewer 2: Otherwise, similar to the membership agreement, you would only have to do one throughout your entire time as an active member. Which, if that's your goal, then I think this can be approved. But I don't remember that being your goal

Reviewer 2: oh we've talked about this before that it means during the academic year.

\asubsubsection{Expectations of Active Members}
Active Members are required to pay dues as stated in \ref{Collection of Dues}, attend all House Meetings and E-Board candidate speeches, and attend at least fifteen of the directorship meetings for each semester in which they are an Active Member.
They are also expected to sign a copy of the Membership Agreement.

> Active Members should also host at least one Technical Seminar during the academic year.

Reviewer: the phrasing of "...should also host..." seems bad. We should say that they are required to/expected to host a seminar

Voting Members are expected to meet the requirements defined in \ref{Expectations of Voting Members}.

\asubsubsection{Expectations of Active Members}
Active Members are required to:
\begin{enumerate}
	\item Pay dues as stated in \ref{Collection of Dues}
	\item Have signed a copy of the Membership Agreement
	\item Attend all House Meetings and E-Board candidate speeches
	\item Attend at least fifteen directorship meetings for each semester in which they are an Active Member

> 	\item Participate in at least one Major Project as defined in \ref{Major Project}

Reviewer 2: ```suggestion
	\item Participate in at least one major project as defined in \ref{Major Project}
```
I believe this should be lower case

> 	\item Host at least one Technical seminar or pass a Technical \ref{Major Project}

Reviewer 2: ```suggestion
	\item Host at least one Technical seminar or pass at least one Technical major project as described in \ref{Major Project}
```
otherwise you'll have "or pass a Technical 4.B.3.A.1"
See how the constitution handles refs for examples of proof

Author: Thank you, that would have been bad lol

Author: Oh crap that means I broke it everywhere else. Hang on

Voting Members are expected to meet the requirements defined in \ref{Expectations of Voting Members}.

\asubsubsection{Expectations of Active Members}
Active Members are required to:
\begin{enumerate}
	\item Pay dues as stated in \ref{Collection of Dues}
	\item Attend all House Meetings and E-Board candidate speeches
	\item Attend at least fifteen directorship meetings for each semester in which they are an Active Member.

> 	\item Sign a copy of the Membership Agreement.

Reviewer: Nitpick: this should be the last item in the list, or right after paying dues

Author: What should be the last item in the list?

Reviewer: The membership agreement one

Author: Oh. Sorry GitHub did not show me line number at first, and then i refreshed twice and it did. :/

Reviewer 2: Personally I think it should go shortest to longest phrase for prettiness

Reviewer 2: but that's preference, not correctness

Author: fixed!

Reviewer 2: You're missing some periods in this section

Reviewer 2: Removing periods is also acceptable

	\item Host at least one Technical seminar or pass a Technical major project
\end{enumerate}

> \\* \\*

Reviewer 2: ```suggestion
```

causes the makefile to get confused. I don't think we need the line there but if you want it you can use vspace or leave a blank line inbetween

> Active Members must participate in at least one major project during the academic year.

Reviewer 2: needs technical

Reviewer 2: Okay well this didn't listen. The line AFTER the enumerate needs technical

Reviewer 2: it's showing up correctly on review file though idk whats wrong with my internet or wtv

Author: Oh I understand what you mean. I see the bug, but the bug is not that Technical needs added. MPs don't have to be Technical after all.

Voting Members are expected to meet the requirements defined in \ref{Expectations of Voting Members}.

\asubsubsection{Expectations of Active Members}
Active Members are required to:
\begin{enumerate}
	\item Pay dues as stated in \ref{Collection of Dues}
	\item Have signed a copy of the Membership Agreement
	\item Attend all House Meetings and E-Board candidate speeches
	\item Attend at least fifteen directorship meetings for each semester in which they are an Active Member
	\item Host at least one Technical seminar or pass a Technical major project
\end{enumerate}

> Active Members must participate in at least one major project during the academic year.

Reviewer 3: Should this be a part of the list?

There's a lot of text related to this one but it feels weird to say "here's all the stuff you should do" anyway "also you have to do this"

Author: I think I made this more clear, let me know!

	\item To report pertinent information to members at the following House Meeting
	\item To maintain records of the goals defined by each previous E-Board
	\item To act as a Judicial Board as defined in \ref{Judicial}

> 	\item To review Major Projects as defined in \ref{Major Project}

Reviewer 2: ```suggestion
	\item To review major projects as defined in \ref{Major Project}
```
see above

> 	\item To review \ref{Major Project}s as defined in \ref{Expectations of Active Members}

Reviewer 2: ```suggestion
	\item To review major projects as defined in \ref{Major Project}
```

See above

Reviewer 2: The ref should still be Major Project, as major projects are defined in their own section now. Lmk if you think otherwise

Reviewer 2: Also resolving comments should really be done by a reviewer, so that other people trying to approve can see if things were actually fixed sufficiently. I get it's easy to see it as a checklist but it really does help reviewers

	\item To oversee the planning, organization, and construction of technical projects for the CSH's benefit
	\item To strive to fulfill the CSH's need for technical equipment
	\item To organize and support Technical Seminars

> 	\item To approve Technical major projects

Reviewer: This makes it sound like RnD can unilaterally pass an MP if it is considered Technical

Author: To approve the Technicality of major projects?

Reviewer: Idk, it's better ig but still not a huge fan. Not sure how else to phrase it though.

	\item To exercise general supervision over R\&D operations
	\item To oversee the planning, organization, and construction of technical projects for the CSH's benefit
	\item To strive to fulfill the CSH's need for technical equipment
	\item To organize and support seminars

> 	\item To approve if a major project or seminar is Technical

Reviewer: Still trying to decide how I feel about phrasing here. Input from other people would be great

Reviewer 2: Is what it is now better @pikachu0542 ?

Reviewer: Yes

> 	\item To approve if a major project or seminar is Technical

Reviewer 2: ```suggestion
	\item To determine if a major project or seminar is Technical
```

	\item To exercise general supervision over R\&D operations
	\item To oversee the planning, organization, and construction of technical projects for the CSH's benefit
	\item To strive to fulfill the CSH's need for technical equipment

> <<<<<<< HEAD

Reviewer: You missed a conflict

	\item To exercise general supervision over R\&D operations
	\item To oversee the planning, organization, and construction of technical projects for the CSH's benefit
	\item To strive to fulfill the CSH's need for technical equipment
	\item To organize and support seminars

> 	\item To determine if a Major Project or seminar is Technical

Reviewer 2: ```suggestion
	\item To determine if a major project or seminar is Technical
```
see above

> 	\item To determine if a \ref{Major Project} or seminar is Technical

Reviewer 2: see above
